src/transport/FabricTable.cpp

```diff
@@ -173,13 +173,14 @@ CHIP_ERROR FabricInfo::FetchFromKVS(PersistentStorageDelegate * kvs)
     return err;
 }
 
-CHIP_ERROR FabricInfo::GetCompressedId(const UncompressedPeerId & uncompressedPeerId, PeerId & compressedPeerId) const
+CHIP_ERROR FabricInfo::GetCompressedId(const RawPeerId & rawPeerId, PeerId * compressedPeerId) const
```

Comment:
When I suggested making the argument clearer it's an output, I didn't think changing away from reference was necessary, but maybe renaming `outCompressedPeerId` or such. Making it a pointer also helps though, so this is satisfactory as-is I would say.